diff --git a/node_modules/react-native-maps/lib/components/MapPolyline.js b/node_modules/react-native-maps/lib/components/MapPolyline.js
index d162654..6bcbb5e 100644
--- a/node_modules/react-native-maps/lib/components/MapPolyline.js
+++ b/node_modules/react-native-maps/lib/components/MapPolyline.js
@@ -132,9 +132,7 @@ const propTypes = {
 
 const defaultProps = {
   strokeColor: '#000',
-  strokeWidth: 1,
-  lineJoin: 'round',
-  lineCap: 'round',
+  strokeWidth: 1
 };
 
 class MapPolyline extends React.Component {
